### PR TITLE
Intialize project with component

### PR DESCRIPTION
**Pull request by gemmen29**
In this PR I add the following: 

1. Initialize React app.
2. Add React Redux.
3. Follow the duck pattern.
4. Add React Router.
5. Add two roots and links.
6. Add books component.
7. Add categories component.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,16 +6,25 @@
     "@testing-library/jest-dom": "^5.16.1",
```
Comment (ZeenatLawal) [inline package.json:7]: - [ ] Kindly write a descriptive README following the Microverse [template](https://github.com/microverseinc/readme-template).

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from './Header.module.css';
+
+const Header = () => (
+  <header>
+    <nav className={styles.nav}>
+      <h1>BookStore</h1>
+      <NavLink to="/books">Home</NavLink>
+      <NavLink to="/categories">Calculator</NavLink>
```
Comment (ZeenatLawal) [inline src/components/Header/Header.js]: - [ ] The text for this link should be Categories, there seems to be a typo. Kindly modify this.

File: src/components/Books/BookItem/BookItem.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './BookItem.module.css';
+
+const BookItem = ({ book }) => (
+  <div className={styles.BookItem}>
+    {`Name: ${book.name}, Category: ${book.category}`}
+  </div>
+);
```
Comment (ZeenatLawal) [inline src/components/Books/BookItem/BookItem.js]: - [ ] The BookItem component should also have a remove button(no functionality yet) as specified in the requirements.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @gemmen29, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- Git flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @gemmen29,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**